README.md

```diff
@@ -5,6 +5,8 @@
 ## Keysy-peasy
 Keysy-peasy makes keyboard shortcuts easy peasy.
 
+**Please note - Keysy-peasy does not handle browser differences for `key` values, eg, IE uses "Esc" whereas other
+browsers use Escape**
```

Comment:
No quite sure why the new line is needed 🤔. Typically IDEs and Markdown viewers will wrap the lines for you (if it gets too long).